packages/accounts/package.json

```diff
@@ -75,6 +75,7 @@
   "gitHead": "ee46e8bb857de3b631044fa70714ea706d9e317d",
   "dependencies": {
     "@alchemy/aa-core": "^3.18.2",
+    "@alchemy/aa-alchemy": "^3.18.2",
```

Comment:
This is here for local dev purposes. I'll remove it when moving this out of draft status.